src/redirect.cc

```diff
@@ -180,7 +182,9 @@ static void
 storeIdHandleReply(void *data, const Helper::Reply &reply)
 {
     RedirectStateData *r = static_cast<RedirectStateData *>(data);
-    debugs(61, 5,"StoreId helper: reply=" << reply);
+    debugs(61, 5, "reply=" << reply);
+    if (reply.expires.has_value())
+        debugs(61, 5, "ignoring unexpected ttl=" << (reply.expires.value() - current_time.tv_sec));
```

Comment:
Should we warn the admin (at debug level 1, possibly with a FadingCounter) that their helper is sending TTLs that Squid is ignoring?